modules/administration/pages/space-management.adoc

```diff
@@ -0,0 +1,67 @@
+[[space-management]]
+= Managing Disk Space
+
+Running out of disk space can have a severe impact on the {productname} database and file structure which, in most cases, is not recoverable.
```

Comment:
Maybe "in some cases" instead of "most cases". I think postgresql also reserves some space to avoid running out of space, but it is good to make people aware that such a condition should be avoided by all means.